src/net/http/Request.cpp

```diff
@@ -41,6 +41,11 @@ Net::Request::Request(Session &_session, ResponseHandler &_handler,
   handle.SetUserAgent(user_agent);
   handle.SetFailOnError(true);
   handle.SetWriteFunction(WriteCallback, this);
+#if defined(_WIN32)
+  // otherwise under Windows it is no access to https urls per SSL
+  // maybe the better option is to adjust the openssl component!
```

Comment:
No, we should fix the underlying issue instead of doing such a kludge.

Reply:
This needs much more investigations, I thought, a workaround is better as a complete failure: In the moment under Windows XCSoar has NO access to any https page yet